tests/commands/test_commands.py

```diff
@@ -35,6 +35,30 @@ def test_setup_utils_configuration():
     assert config['exchange']['secret'] == ''
 
 
+def test_cfg_argument_replaces_configuration():
+    args = [
+        'list-exchanges', '--config', 'config_bittrex.json.example',
+    ]
+
+    # Check original config
+    config = setup_utils_configuration(get_args(args), RunMode.OTHER)
+    assert config['dry_run'] is True
+    assert config['exchange']['key'] == ''
+    assert config['stake_amount'] == 0.05
+    assert config['timeframe'] == '5m'
+
+    args = [
+        'list-exchanges', '--config', 'config_bittrex.json.example',
+        '--cfg', 'stake_amount:float=0.1', 'timeframe=15m', 'exchange.sandbox:bool=true',
```

Comment:
While the syntax may seem simple for a developer / technically versed user, it'll be VERY difficult to understand and apply for non-technical users (which we have a TON - so we MUST ensure it's easy to use for them).

Also, the error-messages are very unclear and missleading.

for example: 
`--cfg stake_amount=float=0.1`  results in `freqtrade trade: error: argument --cfg: could not parse argument "stake_amount=float=0.1" as k=v format`, which however is not the right format for this key, and following the guidance of the message (`--cfg stake_amount=0.1`) i get a different error (`jsonschema.exceptions.ValidationError: '0.1' does not match 'unlimited'`).